### ISSUE TITLE
Edna & Harvey: The Breakout (255320)

### ISSUE THREAD

**Issue by veikk0** [ValveSoftware/Proton#1618]
# Compatibility Report
- Name of the game with compatibility issues: Edna & Harvey: The Breakout
- Steam AppID of the game: 255320

## System Information
- GPU: Intel HD Graphics 3000
- Driver/LLVM version: 3.0 Mesa 18.0.5
- Kernel version: 4.15.0-34-generic
- Link to full system information report: https://gist.github.com/veikk0/771baf36975a0f5be80c59ebafd29efe
- Proton version: 3.7-6

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
After installing four dependencies/redistributables, the game attempts to launch but fails to do so with no GUI errors of any kind. When attempting to launch again by pressing the Play button, the game re-installs two redistributables, the first of which was Java JRE 1.8 and the second one went by so quickly I didn't see the name. The game still fails to launch. This Java-reinstalling behavior repeats on subsequent launch attempts.

Proton log for the first launch attempt: 
[steam-255320-first.log](https://github.com/ValveSoftware/Proton/files/2422487/steam-255320-first.log)
Proton log for the second launch attempt: 
[steam-255320-second.log](https://github.com/ValveSoftware/Proton/files/2422490/steam-255320-second.log)

## Reproduction
Download and run the game.

**Comment by kisak-valve** [comment]
For reference: `err:module:LdrInitializeThunk Main exe initialization for` ... `failed, status c0000017`

**Comment by zzhiyi** [comment]
0026:err:mscoree:CLRRuntimeInfo_GetRuntimeHost Wine Mono is not installed

**Comment by JakobDev** [comment]
The game launches with Proton 4.2 but only in Window Mode. Fullscreen mode is not working.

**Comment by madewokherd** [comment]
Testing on 4.11 branch + Wine Mono from master: I get a crash in System.WriteLine, which I can work around by hacking Console.Open to always return Stream.Null (thus disabling all console I/O). After that, I get a dialog saying "Java Runtime Environment 8 not found."

Fullscreen vs windowed doesn't make any difference.

**Comment by madewokherd** [comment]
The JRE error seems to be correct - the prefix doesn't have any files with "java" or "jre" in the name.

**Comment by madewokherd** [comment]
Manually running jre-8u191-windows-i586.exe from the game's directory shows an msi error.

**Comment by RiantAnda** [comment]
The game is working for me with Proton 5.0-8, however without sound. 
"OpenGL HW acce." must be disabled in game menu for the cursor to work.

The log shows issues with openal?
`17487.275:00e4:00ec:trace:loaddll:free_modref Unloaded module L"C:\\windows\\system32\\OpenAL32.dll" : builtin
17487.277:00e4:00ec:trace:loaddll:load_so_dll Loaded L"C:\\windows\\OpenAL32.dll" at 0x848e0000: builtin
mmap() failed: Cannot allocate memory
Failed to create permanent mapping for memfd region with ID = 1075988749
mmap() failed: Cannot allocate memory
Failed to create permanent mapping for memfd region with ID = 3226135629
mmap() failed: Cannot allocate memory
Failed to create permanent mapping for memfd region with ID = 1075988749
mmap() failed: Cannot allocate memory
Failed to create permanent mapping for memfd region with ID = 3508405029
ALSA lib pcm_dmix.c:1089:(snd_pcm_dmix_open) unable to open slave`
[steam-255320.log](https://github.com/ValveSoftware/Proton/files/4742721/steam-255320.log)

**Comment by madewokherd** [comment]
You might want to try setting openal32 to native. It's not part of Windows, so there's no real need for builtin to exist anyway.

**Comment by RiantAnda** [comment]
@madewokherd 
Thanks for the info. I tried it and got rid of those OpenAL errors, but still no audio
[steam-255320.log](https://github.com/ValveSoftware/Proton/files/4747787/steam-255320.log)

**Comment by madewokherd** [comment]
Looking at the log, it's trying the pulseaudio driver first, then giving up on that and trying ALSA, which also fails.

Does audio work in Proton or Wine otherwise?

**Comment by RiantAnda** [comment]
Nope, I have no audio issues with other Proton games, except [daedalic games](https://github.com/ValveSoftware/Proton/issues/1412), which is strange...
So you are getting audio in this game?

**Comment by RiantAnda** [comment]
The game engine changed to Unity on Feb 08, 2021. This game runs now without issues for me.

Maybe somebody could check with #3308.  This version of the game had the Unity engine from the beginning and should perform better now with modern Proton?
